src/Compilers/CSharp/Portable/Symbols/AssemblySymbol.cs

```diff
@@ -305,6 +305,10 @@ public sealed override Symbol ContainingSymbol
             }
         }
 
+        internal abstract bool HasGuidAttribute { get; }
```

Comment:
>HasGuidAttribute [](http://example.com/codeflow?start=31&length=16)

It looks like we already have `GetGuidString` method. How is this one different?